fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/ContainmentIndexImpl.java

```diff
@@ -275,12 +288,13 @@ public void removeContainedBy(final Transaction tx, final FedoraResource parent,
     }
 
     @Override
+    @Transactional
```

Comment:
Is Spring's TransactionManager already enabled?

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/transaction/annotation/EnableTransactionManagement.html